modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -83,11 +83,74 @@ In your network, you might need to whitelist the hostnames and ports of external
 [[install_Istio]]
 == Installing Istio
 
-You can install Istio 1.4.x or 1.5.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
+You can install Istio 1.6.x, 1.5.x, or 1.4.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
+
+=== Install Istio 1.6.x
+
+Anypoint Service Mesh supports Istio versions `1.6.0` through `1.6.8`. To install and configure Istio 1.6.x for Anypoint Service Mesh:
+
+. Download Istio using the https://istio.io/docs/setup/[Istio documentation].
+. Install Istio with the following configurations:
+.. Enable the policy control flag:
++
+[source,text,linenums]
+----
+values:
+  meshConfig:
+    disablePolicyChecks: false
+  components:
+    policy:
+      enabled: true    
+----
+.. Enable telemetry:
++
+[source,text,linenums]
+----
+values:
+  telemetry:
+    v1:
+      enabled: true
+    v2:
+      enabled: false
+components:
+  citadel:
+    enabled: true
+  telemetry:
+    enabled: true
+----
+The following example illustrates a full manifest: 
++
+[source,text,linenums]
+----
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  profile: default
+  components:
+    policy:
+      enabled: true
+    telemetry:
+      enabled: true
+    ingressGateways:
+    - enabled: false
+  meshConfig:
+    disablePolicyChecks: false
+  values:
+    telemetry:
+      v1:
+        enabled: true
+      v2:
+        enabled: false
+----  
+When using the full manifest, ensure that you choose the correct profile. In the example, the `default` profile is used. 
+
+To install the manifest, run:
++
+`istioctl manifest apply -f <manifest-file.yaml>`
 
 === Install Istio 1.5.x
 
-Anypoint Service Mesh supports Istio versions `1.5.0` through `1.5.2`. To install and configure Istio 1.5.x for Anypoint Service Mesh:
+Anypoint Service Mesh supports Istio versions `1.5.0` through `1.5.9`. To install and configure Istio 1.5.x for Anypoint Service Mesh:
```

Comment:
ditto same edits made to similar sentence for 1.6.x

Reply:
Nope the supported versions are as have been documented. Done! Thanks!